core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/expressions.scala

```diff
@@ -4,6 +4,10 @@ import java.util.{Locale, UUID}
 
 // Expression used to refer to fields, functions and similar. This can be used everywhere
 // expressions in SQL appear.
+case class ExpressionLogicalPlan(expression: Expression) extends LeafNode {
+  override def output: Seq[Attribute] = Seq.empty
+}
+
```

Comment:
```suggestion
```
no need to wrap expression, it's unnecessary.

Reply:
Fixed

Reply:
we have `ScalarSubquery` for a think exactly opposite to this, just in case.

Reply:
That's a good direction, let me see if that helps.